packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Axes/index.tsx

```diff
@@ -0,0 +1,267 @@
+                        <AxisMinMax
+                            label={`Auto ${
+                                dirtyLayout?.flipAxes ? 'y' : 'x'
+                            }-axis range`}
+                            min={dirtyEchartsConfig?.xAxis?.[0]?.min}
+                            max={dirtyEchartsConfig?.xAxis?.[0]?.max}
+                            setMin={(newValue) => setXMinValue(0, newValue)}
+                            setMax={(newValue) => setXMaxValue(0, newValue)}
+                        />
+                    )}
+                    <Group spacing="xs">
+                        <Group spacing="xs">
+                            <Config.SubLabel>Sort</Config.SubLabel>
+                            <SegmentedControl
+                                defaultValue={
+                                    dirtyEchartsConfig?.xAxis?.[0]?.inverse
+                                        ? 'descending'
+                                        : 'ascending'
+                                }
+                                data={[
+                                    {
+                                        value: 'ascending',
+                                        label: (
+                                            <MantineIcon
+                                                icon={IconSortAscending}
+                                            />
+                                        ),
+                                    },
+                                    {
+                                        value: 'descending',
+                                        label: (
+                                            <MantineIcon
+                                                icon={IconSortDescending}
+                                            />
+                                        ),
+                                    },
+                                ]}
+                                onChange={(value) => {
+                                    setInverseX(value === 'descending');
+                                }}
+                            />
+                        </Group>
+                        {!dirtyLayout?.flipAxes && (
+                            <Group noWrap spacing="xs" align="baseline">
+                                <Config.SubLabel>Rotation</Config.SubLabel>
+                                <NumberInput
+                                    type="number"
+                                    defaultValue={
+                                        dirtyEchartsConfig?.xAxis?.[0].rotate ||
+                                        0
+                                    }
+                                    min={0}
+                                    max={90}
+                                    step={15}
+                                    maw={54}
+                                    rightSection="°"
+                                    onChange={(value) => {
```

Comment:
Some of these inputs are onChange and some are onBlur. It can be a little confusing to use. 

But now that I play around with it, this is true in main as well. I guess maybe that can be food for future thought.

Reply:
That's true. I'm going to have a look at every single section where we do an `onBlur` and sections where I've changed from `onBlur` to `onChange` and figure out what's best 👍

Reply:
And here it is! https://github.com/lightdash/lightdash/pull/9528/commits/cc6befd3fae4268f9678a909e327b91ded160fe3